index.html

```diff
@@ -15,16 +15,21 @@
         <h1><span>K</span>-Kal</h1>
       </a>
     </div>
+    <button id="open" class="open-menu" ><i class="fa-solid fa-bars"></i></button>
     <nav class="nav">
-      <div class="nav-container">
-        <ul class="side-bar" id="menu">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#">Contact</a></li>
-        </ul>
-        <a href="#menu" class="nav-menu"><i class="fa-solid fa-bars nav-icon"></i></a>
-        <a href="#" class="nav-menu nav-menu-second"><i class="fa-solid fa-x nav-icon"></i></a>
-      </div>
+      <button id="close" class="close-menu"><i class="fa-solid fa-x"></i></button>
+      <ul class="side-bar">
+        <li class="nav-item"><a class="nav-link" href="#portfolio">Portfolio</a></li>
+        <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
+        <li class="nav-item"><a class="nav-link" href="#contact">Contact</a></li>
+      </ul>
```

Comment:
- [ ] There is an issue with your design. One of the requirements is that when the user clicks on any mobile-menu link, the mobile menu should disappear. you can achieve that by grabbing this `side-bar` in javascript and then implementing a forEach on that, where you will place the addEventListener on each element of that forEach and remove this class `nav.classList.remove('visible');` in the click event.
requirement: 
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.